.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: lint
```

Comment:
Is the workflow automatically started, or does it need to be referenced somewhere?

Reply:
On Line 3, it will start on any push event